lib/tasks/data_migrations/02_import_articles_into_news_items.rake

```diff
@@ -0,0 +1,24 @@
+require_relative '../batch_process'
+
+namespace :data_migrations do
+  task :import_articles_into_news_items => :environment do
+    old_article_feed_source = FeedSource.find_or_create_by(
+      name: 'Old Coinfi Articles',
+      is_active: false,
+      feed_url: '',
+      site_url: '',
+    )
+
+    batch_process(Article.all) do |article|
+      NewsItem.create!(
+        feed_source: old_article_feed_source,
+        feed_item_id: article.url,
+        url: article.url,
+        title: article.title,
+        summary: article.summary,
+        feed_item_published_at: article.published_date,
+        feed_item_updated_at: article.updated_at,
+      )
+    end
```

Comment:
@gduquesnay Do we also need to add a line to set the `is_active` field to true for all `FeedSources`?

Reply:
We may also want to note in a comment that this data migration should be run after running `20180524092136`.

Reply:
@hanchang  I usually have rails db:migrations automated but these data migration have to be run manually post deploy so I can usually depend on the db:migration(s) they depend on to have already been run.  Is that not the case here?